README.md

```diff
@@ -119,26 +120,38 @@ configuration will send messages to the test apel server.
 
 ## Adding Files
 
-There are two ways to add files to be sent:
+There are multiple manual and programmatic ways to add files to be sent: 
 
 ### Manual
 
+#### With the dirq module
 All file and directory names must use hex characters: `[0-9a-f]`.
 
  * Create a directory within /var/spool/apel/outgoing with a name
    of EIGHT hex characters e.g. `12345678`
  * Put files in this directory with names of FOURTEEN hex 
    e.g. `1234567890abcd`
```

Comment:
I know you didn't touch this, but do the first eight characters have to match those in the directory name?

Reply:
No, they're independent: https://dirq.readthedocs.io/en/latest/queue.html#directory-structure

Reply:
That's what I thought, but then the example uses the same characters...